rpc_jobs/rpc_aio.yml

```diff
@@ -329,21 +329,20 @@
                   ]
                 aio_prepare.prepare()
                 deploy.deploy_sh(environment_vars: environment_vars)
-                tempest.tempest()
+                tempest.tempest_install()
```

Comment:
I'm against this. The purpose is to verify that the base deployment was successful. If tempest is failing all the time we should address that and not ignore it.